### DISCUSSION TITLE
Keyboard light color per profile?

### DISCUSSION THREAD

**Discussion by MattDevOps** [seerge/g-helper#2419] [Q&A]
Is it possible to set the keyboard light color per profile? I made a new ghelper profile and want the keyboard to only to switch to a certain color on this profile, and back to the other color on the other profile. Ty

**Comment by seerge** [comment]
@MattDevOps Modes (Profiles?) are only about performance (i.e. fan settings / limits / etc), you can't assign keyboard settings to modes.
